modules/porous_flow/doc/content/modules/porous_flow/solute_tracer_transport.md

```diff
@@ -0,0 +1,153 @@
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=FluidProperties
+
+### Variables declaration
+
+Then, we need to declare what type of unknown we are solving. This can be done in the variable code block. Since we are focusing on the concentration of tracer at x =50 m, C is a desired variable. Porepressure is also used for the fluid motion. Since we only have constant initial conditions so they can be directly supplied here.
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=Variables
+
+### Kernel declaration
+
+This section is to tell the solver what problem we need to solve. To do so, some kernels are declared. In MOOSE, the required kernels depend on the terms in the governing equations. For this problem, six kernels were declared. To have a better understanding, users are recommended to visit [this page](https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html). The code block is shown below with the first three kernels associated with equation 1  and the remain associated with equation 2.
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i start=[Kernels] end=[FluidProperties]
+
+
+### Material setup
+
+Since these kernels require more inputs to feed into. Thus, we need to add them following the requirements. Those inputs are declared here:
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=Materials
+
+### Boundary Conditions
+
+The next step is to supply the boundary conditions. There are three Dirichlet boundary conditions that we need to supply. The first two are constant pressure at the inlet and outlet denoted as "constant_inlet_pressure" and "constant_outlet_porepressure". Another one is constant tracer concentration at the inlet denoted as "inlet_tracer". Finally, a PorousFlowOutflowBC boundary condition was supplied at the outlet to simulate the tracer moving out of the domain.
+
+!listing modules/porous_flow/examples/solute_tracer_transport/solute_tracer_transport.i block=BCs
+
+### Executioner setup
+
+Lastly, we will tell MOOSE the type of the problem and how we want it to perform or so call Executioner setup (shown in the code block below). For this problem, a transient solver is required.
+To save time and computational resources, "IterationAdaptiveDT" was implemented. This option enables the time step to be increased if the solution converges and decreased if it does not. Thus, this will help save time if a large time step can be used and aid in convergence if a small time step is required [(Details can be found here)](https://mooseframework.inl.gov/source/timesteppers/IterationAdaptiveDT.html). For practice, users can try to disable it by putting the code block into comment and witnessing the difference in the solving time and solution.
```

Comment:
```suggestion
To save time and computational resources, [`IterationAdaptiveDT`](IterationAdaptiveDT.md) was implemented. This option enables the time step to be increased if the solution converges and decreased if it does not. Thus, this will help save time if a large time step can be used and aid in convergence if a small time step is required. For practice, users can try to disable it by putting the code block into comment and witnessing the difference in the solving time and solution.
```